src/mono/wasm/debugger/BrowserDebugProxy/MonoSDBHelper.cs

```diff
@@ -709,26 +709,101 @@ public FieldTypeClass(int id, string name, int typeId, bool isNotPrivate, FieldA
     }
     internal class ValueTypeClass
     {
-        public byte[] valueTypeBuffer;
-        public JArray valueTypeJson;
-        public JArray valueTypeJsonProps;
-        public int typeId;
-        public JArray valueTypeProxy;
-        public string valueTypeVarName;
-        public bool valueTypeAutoExpand;
-        public int Id;
-        public ValueTypeClass(string varName, byte[] buffer, JArray json, int id, bool expand_properties, int valueTypeId)
-        {
-            valueTypeBuffer = buffer;
-            valueTypeJson = json;
-            typeId = id;
-            valueTypeJsonProps = null;
-            valueTypeProxy = null;
-            valueTypeVarName = varName;
-            valueTypeAutoExpand = expand_properties;
+        private JArray Json { get; }
+        private int TypeId { get; }
+        private bool AutoExpand { get; }
+        private int Id { get; }
+        private JArray Proxy { get; set; }
+        private JArray JsonProps { get; set; }
```

Comment:
1. These are private, and used within the class, so they don't need to be properties as we are not hiding them, or protecting them. So, we can make them fields
2.  And field names shouldn't start with a capital letter
3. Some of these are only set in the constructor, so they should be marked `readonly`.

```suggestion
        private readonly JArray json;
        private readonly int typeId;
        private readonly bool autoExpand;
        private readonly int id;
        private JArray proxy;
        private JArray jsonProps;
```